jobboard/migrations/0004_auto_20221220_2325.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.16 on 2022-12-20 23:25
```

Comment:
I'd delete all the migrations and create the initial again - that way this goes out with just 1 migration. Bit cleaner that way.